src/electionguard/ballot.py

```diff
@@ -367,6 +367,19 @@ def __ne__(self, other: Any) -> bool:
         return not self.__eq__(other)
 
 
+@dataclass
+class CiphertextContest(ElectionObjectBase):
+    """
+    Encrypted contest
```

Comment:
can we add some more detail here?

Reply:
Yeah. It looks like it got cut off.

Reply:
If you look at CiphertextSelection, this lines up 1 to 1.